pkg/server/node.go

```diff
@@ -195,6 +195,18 @@ This metric is thus not an indicator of KV health.`,
 		Measurement: "Bytes",
 		Unit:        metric.Unit_BYTES,
 	}
+	metaActiveRangeFeed = metric.Metadata{
+		Name:        "stream.rangefeed.count",
```

Comment:
How about an `rpc.streams.` prefix for consistency with other RPC metrics, and have both a counter and a gauge for each method?

We could also consider generating automatic metrics for all streaming methods, similarly to `rpc.method.%s.recv` for regular requests. But we can punt that as long as the metrics scheme is extensible.